### PR TITLE
feat: added verifier capability

### PR DESCRIPTION
**Pull request by Artemkaaas**
Changes:
* Applied BC Wallet styles to Verifier related screens and components
* Defined proof request templates according to https://xd.adobe.com/view/993bddd5-51cd-435b-b1bf-6b2224de6d99-cc68/screen/1a48e9dd-39df-4781-bcea-0b1a8a602e09/
* Updated `useNotifications` hook to fetch proof records
* Updated developer options screens to support verifier and connection inverter capabilities

### CODE REVIEW

**Comment by jleach** [comment]
@Artemkaaas Able to add a brief description to this PR?

**Review by bryce-mcmath — DISMISSED** [review]

File: app/src/constants.ts
Comment (jleach) [inline app/src/constants.ts:1]: I feel like constants.ts isn't the right place for these templates. Can you move them to a `templates.ts` file?

**Review by jleach — CHANGES_REQUESTED** [review]

**Comment by jleach** [comment]
@Artemkaaas Able to make the one change requested and fix conflicts so we can then this one merged.

File: app/src/constants.ts
Comment (Artemkaaas) [reply app/src/constants.ts:1]: done

**Comment by Artemkaaas** [comment]
> @Artemkaaas Able to make the one change requested and fix conflicts so we can then this one merged.

Done

**Comment by jleach** [comment]
@Artemkaaas Looking better. Build failed with the following:

```console
Error: src/index.ts(25,39): error TS2307: Cannot find module './constants' or its corresponding type declarations.
ERROR: "build:app" exited with 2.
ERROR: "build:all" exited with 1.
npm ERR! code 1
npm ERR! path /Users/runner/work/bc-wallet-mobile/bc-wallet-mobile
npm ERR! command failed
npm ERR! command sh -c run-s install:all build:all

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/runner/.npm/_logs/2023-04-13T21_19_29_545Z-debug-0.log
Error: Process completed with exit code 1.
```

**Review by jleach — APPROVED** [review]

**Review by bryce-mcmath — APPROVED** [review]
Nice work!

File: app/src/request-templates.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    version: '0.0.1',
+    payload: {
+      type: ProofRequestType.Indy,
+      data: [
+        {
+          schema: 'XUxBrVSALWHLeycAUhrNr9:2:Person:1.0',
+          requestedAttributes: [
+            {
+              name: 'given_names',
+              restrictions: [{ schema_id: 'XUxBrVSALWHLeycAUhrNr9:2:Person:1.0' }],
```
Comment (wadeking98) [inline app/src/request-templates.ts:18]: We need to make all the restrictions by credential definition id, not schema id. Anyone can create a credential with a given schema_id, so anyone can issue themselves a credential that passes this proof request.

**Review by wadeking98 — CHANGES_REQUESTED** [review]

File: app/src/request-templates.ts (hunk 1 above)
Comment (wadeking98) [reply app/src/request-templates.ts:18]: going to approve this to get it in the codebase, but I'll create a separate issue for changing the restrictions

**Review by wadeking98 — APPROVED** [review]

**Pull request merged**